src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let previous = initialValue;
```

Comment:
Use more descriptive names.
It's not `previous` value. It's accumulator which have sum of all prev actions